app/src/main/res/values-it/strings.xml

```diff
@@ -5,23 +5,27 @@
     <string name="artists">Artisti</string>
     <string name="albums">Album</string>
     <string name="playlists">Playlist</string>
-
+    <string name="explore">Esplora</string>
+    
     <!-- Top bar -->
     <plurals name="n_selected">
         <item quantity="one">%d selezionato</item>
         <item quantity="other">%d selezionati</item>
-        <item quantity="many">%d selezionati</item>
     </plurals>
 
     <!-- Home -->
     <string name="history">Cronologia</string>
     <string name="stats">Statistiche</string>
     <string name="mood_and_genres">Mood e generi</string>
-    <string name="account">Account</string>
+    <string name="account">Profilo</string>
     <string name="quick_picks">Scelte rapide</string>
     <string name="quick_picks_empty">Ascolta alcuni brani per generare le tue scelte rapide</string>
     <string name="new_release_albums">Nuovi album in uscita</string>
```

Comment:
Can you update new_release_albums to not mention albums? Check the English string.

#276

Reply:
I should have fixed this, @lucasmz-dev

Reply:
DW it wasn't highlighted properly hence why #276 exists, but it's nice to highlight so that you don't have to do this again later :P